app/models/concerns/manual_parts.rb

```diff
@@ -9,6 +9,13 @@ def title
     def page_title
       "#{breadcrumb} - #{manual_page_title}"
     end
+
+    def document_heading
```

Comment:
Can we include a link to the audit trail from government-frontend to the description in commit https://github.com/alphagov/frontend/pull/5027/commits/d26a7929a8a434fcc8455a7c361313e9052fd3f8